### PR TITLE
Add Django sample

### PR DESCRIPTION
**Pull request by tolusha**
Signed-off-by: Anatoliy Bazko <abazko@redhat.com>

### What does this PR do?
Adds django sample [1] to the list of available sample for Che 7 Python stack.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/12945

[1] https://github.com/gothinkster/django-realworld-example-app

### CODE REVIEW

**Comment by tolusha** [comment]
/cc @slemeur 
Know issue: django admin page can't be shown in preview widget, probably due to iframe.

**Comment by tolusha** [comment]
ci-test

**Comment by tolusha** [comment]
It is impossible to test provided commands due to https://github.com/eclipse/che-theia/issues/155

**Review by ashumilova — APPROVED** [review]

File: ide/che-core-ide-stacks/src/main/resources/stacks.json [hunk 1]
```diff
@@ -2576,32 +2576,38 @@
...
         "default": {
           "recipe": {
-            "content": "python:3.7.3",
+            "content": "centos/python-36-centos7:1",
             "type": "dockerimage"
           },
           "machines": {
             "python": {
+              "servers": {
+                "django": {
```
Comment (l0rd) [inline ide/che-core-ide-stacks/src/main/resources/stacks.json:2601]: Do we really need a server? Isn't the [port plugin](https://github.com/eclipse/che-theia/tree/master/plugins/ports-plugin) exposing it automatically?

**Review by l0rd — APPROVED** [review]

File: ide/che-core-ide-stacks/src/main/resources/stacks.json (hunk 1 above)
Comment (tolusha) [reply ide/che-core-ide-stacks/src/main/resources/stacks.json:2601]: >  Isn't the port plugin exposing it automatically?

Yes, it is. But after a little application becomes unavailable. 
So, that's why I added sever to the list

Comment (l0rd) [reply ide/che-core-ide-stacks/src/main/resources/stacks.json:2601]: Are you saying that the application is unstable when using the port plugin but works just fine if you define a server?

Comment (tolusha) [reply ide/che-core-ide-stacks/src/main/resources/stacks.json:2601]: Exactly

Logs from theia-ide container (minishift):
```
root ERROR Uncaught Exception:  Error [ERR_IPC_CHANNEL_CLOSED]: Channel closed
--
  | root ERROR Error [ERR_IPC_CHANNEL_CLOSED]: Channel closed
  | at ChildProcess.target.send (internal/child_process.js:636:16)
  | at HostedPluginProcess.onMessage (/home/theia/node_modules/@theia/plugin-ext/lib/hosted/node/hosted-plugin-process.js:105:31)
  | at ServerPluginProxyRunner.onMessage (/home/theia/node_modules/@eclipse-che/theia-remote/lib/node/server-plugin-proxy-runner.js:49:32)
  | at /home/theia/node_modules/@theia/plugin-ext/lib/hosted/node/hosted-plugin.js:99:28
  | at Array.forEach (<anonymous>)
  | at HostedPluginSupport.onMessage (/home/theia/node_modules/@theia/plugin-ext/lib/hosted/node/hosted-plugin.js:97:32)
  | at HostedPluginServerImpl.onMessage (/home/theia/node_modules/@theia/plugin-ext/lib/hosted/node/plugin-service.js:152:27)
  | at JsonRpcProxyFactory.onNotification (/home/theia/node_modules/@theia/core/lib/common/messaging/proxy-factory.js:246:36)
  | at /home/theia/node_modules/@theia/core/lib/common/messaging/proxy-factory.js:181:53
  | at handleNotification (/home/theia/node_modules/vscode-jsonrpc/lib/main.js:483:

```

Comment (tolusha) [reply ide/che-core-ide-stacks/src/main/resources/stacks.json:2601]: https://github.com/eclipse/che/issues/13170

**Pull request merged**